modules/govuk_crawler/manifests/config.pp

```diff
@@ -0,0 +1,31 @@
+#
+# [*aws_access_key*] access key for AWS
+# [*aws_secret_key*] secret for chosen AWS key
+#
+# Will be set using environment variables, see:
+# http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-environment
+#
+
+class govuk_crawler::config (
+  $aws_access_key = undef,
+  $aws_secret_key = undef,
+) {
+  $env_name = 's3_sync_mirror'
+
+  file { ["/etc/govuk/${env_name}", "/etc/govuk/${env_name}/env.d"]:
+    ensure  => present,
+    purge   => true,
+    recurse => true,
+    force   => true,
+  }
+
+  file { "/etc/govuk/${env_name}/env.d/AWS_SECRET_ACCESS_KEY":
```

Comment:
We are storing secrets in these files, maybe it needs to enforce the file mode and ownership